packages/inter-protocol/src/price/priceOracleKit.js

```diff
@@ -0,0 +1,124 @@
+import { Fail } from '@agoric/assert';
```

Comment:
this is really a rename https://github.com/Agoric/agoric-sdk/pull/6977/commits/1023f04c5625cf1f8fb4cc894f84dee226229979